_posts/2018-06-11-2018-foreman-survey-analysis.md

```diff
@@ -0,0 +1,406 @@
+---
+layout: post
+title: 2018 Foreman Survey Analysis
+date: 2018-06-11 10:00:00
+author: Greg Sutcliffe
+tags:
+- usage
+- foreman
+- events
+- plugins
+- community
+- survey
+- API
+- virtualization
+- puppet
+- katello
+- pulp
+- content
+- use cases
+---
+
+As with previous years, we ran a Foreman Community Survey in order to give you
+all the opportunity to tell us how we're doing - where it's good, and where
```

Comment:
s/you all/y'all  ;)

Reply:
Please refer to Fig 1b - Geography for the local of the majority of our users :D